java/java.lsp.server/vscode/README.md

```diff
@@ -95,13 +111,19 @@ Some refactorings are two steps with like Override method ... where method to be
 * Assign to variable
 * Generate hashCode/equals
 * Generate toString()
-* Change method signature
 * Surround With refactoring
 * For cycle refactoring
 * try-catch refactoring
 * switch() statement
 * while() cycle
 
+## Organize Imports
+It is possible to organize imports in Java classes. There are several settings to help developers to set this feature up to their expectations. Go to VSCode `Preferences | Settings`  and search for _NetBeans_.
```

Comment:
It is _help do something_ - e.g. without _to_. Again what is the message? I'd assume:
1. organizing imports is possible
2. it works automatically
3. it can be configured

right?

_Out of the box support for organizing imports in Java sources is available. It removes unused imports, groups imports by packages and updates your imports whenever a file is saved. In addition to the defaults, there is a rich set of configuration options...._